src/main/java/com/desiato/puresynth/configurations/AuthTokenFilter.java

```diff
@@ -57,6 +61,10 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
     }
 
     private boolean isPublicEndpoint(String requestURI) {
-        return "/api/login".equals(requestURI) || "/api/user".equals(requestURI);
+        boolean isPublic = antPathMatcher.match("/api/login", requestURI) ||
+                antPathMatcher.match("/api/user", requestURI) ||
+                antPathMatcher.match("/api/user/**", requestURI);
```

Comment:
This is dangerous.

Reply:
yes, I forgot to remove the `/api/user/**` endpoint from this method. 
I inserted it to check its functionality.